sql/core/src/main/java/org/apache/spark/sql/vectorized/ColumnarBatch.java

```diff
@@ -96,16 +90,6 @@ public void setNumRows(int numRows) {
    */
   public int numRows() { return numRows; }
 
-  /**
-   * Returns the schema that makes up this batch.
-   */
-  public StructType schema() { return schema; }
-
-  /**
-   * Returns the max capacity (in number of rows) for this batch.
-   */
-  public int capacity() { return capacity; }
```

Comment:
I agree to remove these fields `schema` and `capacity` from `ColumnarBatch`.
Is it better to prepare APIs to get `schema` and `capacity` from a set of `ColumnVector`s?

Reply:
For `ColumnarBatch` consumers, they don't care about `capacity`, but only `numRows`, 
`capacity` is only needed by column vector builders. Also they don't care about schema/field names, but only the data type of each column.